src/Microsoft.AspNet.Authentication.OpenIdConnect/OpenIdConnectAuthenticationOptions.cs

```diff
@@ -233,10 +245,12 @@ public OpenIdConnectProtocolValidator ProtocolValidator
         public string Scope { get; set; }
 
         /// <summary>
-        /// Gets or sets the AuthenticationScheme used when creating the <see cref="System.Security.Claims.ClaimsIdentity"/>.
+        /// Gets or sets the SignInScheme which will be used to set the <see cref="System.Security.Claims.ClaimsIdentity.AuthenticationType"/>.
```

Comment:
In most cases, it will be present, because the OIDC (just like the other social middleware) doesn't issue any cookie: persistence is always delegated to another middleware.

And BTW, you SHOULDN'T set `TokenValidationParameters.AuthenticationType` using `SignInScheme` but with `AuthenticationScheme`. See my PR: https://github.com/aspnet/Security/pull/168/files#diff-960390b7edd1b3a479fbd7fcfd07fbe8R55